blog/_posts/2017-09-04-clojurians-update.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Clojurian's Update - Working on re-frame's Tracing Tool
+layout: post
+created_at: Mon September 4 2017
+permalink: blog/2017-09-04-clojurians-update.md 
+current: blog
+author: Saskia and Chris
+twitter: saskalii
+categories:
+- student-posts
+---
+
+
+# Really Great Summer of Clojure
+
+It's really exciting to see and feel our progress. In the beginning we were grasping at the huge space of ClojureScript tools and libraries and editors, and now as the last month begins our contributions to re-frame’s tracing tool have already made it easier for developers to debug their applications! There's also a hint of melancholy that the summer of code will be ending in four weeks. Thoughts are starting to take shape about how to address the last third of the project. What do we want to achieve? What would be the ideal outcome? Let's gather some thoughts!
+
+## Learning and Growing
+
+Through a glorious combination of working on re-frame's tracing tool, reading other peoples' open source projects, hanging out with our wonderful mentors and coaches, and tinkering with our own side projects, we gathered experience in programming with ClojureScript and re-frame and learned about langugage specific idioms. What we find most valuable though is the great advice we got from our coaches and our mentor. They all have given us so much input for how to become good programmers, not only technically, but especially in terms of finding a good and efficient workflow, addressing problems on a conceptual level and the importance of clear and constructive communication. 
+
+![image](/img/blog/2017/coaching.jpg)
+<div class="image-credits">coaching with Matt! / Photo by: Chris Sun</div>
+
+We are very thankful to have the opportunity to work together so closely with all of you! A big thank you for all your support!
+
+## Building and Collaborating
+
+We worked on many features for the [tracing tool](https://github.com/Day8/re-frame-trace) with the intention to make it more easily usable. The first things we focused on were improving the filter functionality, making resizing of the window smooth and easy and adding some adjustments to the user interface. Another important addition was saving user data like filters to localstorage to ensure persistence. The traces that are being shown in the panel are still a lot and can feel a bit overwhelming. Our focus for the next weeks is to reduce the amount of traces to the ones that are important. For the end of the summer, we want to compare the tracing tool before and after we started working on it.  
+
+## Community <3
+
+In the third week of our project, we went to **EuroClojure**, which took place in Berlin this year. 
+
+![image](/img/blog/2017/euroclojure.png)
```

Comment:
Thanks for the quick fixes but there is one more issue. Two of these image files are of .jpeg formats and not .png. I thought you'll catch it but no issues. You can fix them in the next commit. :)

Reply:
thank you, I pushed the changes!